samples/sample-05/mvnw

```diff
@@ -0,0 +1,316 @@
+#!/bin/sh
+# ----------------------------------------------------------------------------
```

Comment:
This needs to be executable `chmod u+x` on Linux/MacOS; I can do it during merge.